rollup.config.js

```diff
@@ -21,7 +21,7 @@ export default [
     output: {
       file: 'dist/wavesurfer.cjs',
       format: 'cjs',
-      exports: 'default',
+      exports: 'named',
```

Comment:
What implications does this have for the main WaveSurfer export? Will `require('wavesurfer.js')` work the same?

Reply:
@katspaugh It looks like it is modifying the wavesurfer.cjs file
Instead of `module.exports=blah` it will be `exports.default=blah`

I think this would impact NodeJS apps in having to change their import from:
`const WaveSurfer = require('wavesurfer.js')`
to:
`const WaveSurfer = require('wavesurfer.js').default`

But I could be wrong about that.

Reply:
We should double-check that it’s not a breaking change. Especially for the min.js file which people tend to hot-load.

Reply:
Yeah, we checked and this is a breaking change for .cjs users. This should work fine for min.js users though.

Reply:
I would suggest not to add those exports to the main module then.
Why can’t one just import base plugin directly from dist?

https://unpkg.com/wavesurfer.js@7.6.4/dist/base-plugin.js

Reply:
Eslint could not resolve the exports. The application would run but eslint would crash.

We updated the PR to export `base-plugin` and `dom`. Please take a look at the exports section in the `package.json`.

When there is a major version bump we should look at updating how exports are handled to make it easier to add new exports in the future. :)